### DISCUSSION TITLE
opinion for a one line comment

### DISCUSSION THREAD

**Discussion by tyrionchen** [alvr-org/ALVR#1177] [General]
https://github.com/alvr-org/ALVR/blob/2d8041e52815172e9705e0effc4a00cbca725a36/alvr/server/cpp/platform/win32/OvrDirectModeComponent.cpp#L114

Are both poses in SubmitLayerPerEye_t the same?
May be not, there are two eye in VR devices, Their position and orientation is different.

**Comment by zmerp** [comment]
I just checked, only the orientation matters here and even if mHmdPose has different translation for the two eyes, the orientation is the same.

**Reply by tyrionchen** [reply]
ok, no problem
